src/main/java/com/microsoft/azure/relay/RelayLogger.java

```diff
@@ -144,6 +144,8 @@ private static boolean init() {
 		map.put("writingBytesFailed", "%s: writing bytes failed.");
 		map.put("receivedBytes", "%s: received bytes from remote. Total length: %s");
 		map.put("receivedText", "%s: received text from remote. Total length: %s");
+		map.put("clientWebSocketClosing", "%s: is closing. Close reason: %s");
```

Comment:
>clientWebSocketClosing [](start = 11, length = 22)

Are these grouped by purpose (or some concept you have)? If so, great. If not, consider alphabetizing. 

My personal rule in my head is "Try to use some kind of grouping. (It doesn't have to be alphabetical if something else seems to be better)."

Reply:
Ok, will make it alphabetical. Currently it was mostly grouped by files.

---
In reply to: [257779837](https://github.com/Azure/azure-relay-java/pull/14#discussion_r257779837) [](ancestors = 257779837)

Reply:
You don't have to, I was just making the comment: Consider grouping by some heuristic. If files is it, then that's fine.

---
In reply to: [258614069](https://github.com/Azure/azure-relay-java/pull/14#discussion_r258614069) [](ancestors = 258614069,257779837)